web/controllers/coherence/session_controller.ex

```diff
@@ -0,0 +1,323 @@
+defmodule CoursePlanner.Coherence.SessionController do
```

Comment:
cool, 
adding the session_controller was the only option i have found but it was pretty ugly